codegen/core/src/test/resources/test-files/descriptor-sets/generate-from-samples.sh

```diff
@@ -32,15 +34,29 @@ echo "Generated event-sourced-shoppingcart.desc"
 
 protoc --include_imports \
 --proto_path="$1/protocols/sdk/src/main/protobuf" \
+--proto_path="$2/codegen/core/src/main/protobuf" \
 --proto_path="$2/samples/java-valueentity-shopping-cart/src/main/proto" \
 --descriptor_set_out=value-shoppingcart.desc \
 "$2/samples/java-valueentity-shopping-cart/src/main/proto/com/example/shoppingcart/domain/shoppingcart_domain.proto" \
 "$2/samples/java-valueentity-shopping-cart/src/main/proto/com/example/shoppingcart/shoppingcart_api.proto"
 
 echo "Generated value-shoppingcart.desc"
 
+
+protoc --include_imports \
+--proto_path="$1/protocols/sdk/src/main/protobuf" \
+--proto_path="$2/codegen/core/src/main/protobuf" \
+--proto_path="$2/samples/java-valueentity-shopping-cart/src/main/proto" \
+--descriptor_set_out=action-shoppingcart.desc \
+"$2/samples/java-valueentity-shopping-cart/src/main/proto/com/example/shoppingcart/shoppingcart_controller_api.proto" \
+"$2/samples/java-valueentity-shopping-cart/src/main/proto/com/example/shoppingcart/shoppingcart_api.proto"
+
+echo "Generated action-shoppingcart.desc"
```

Comment:
We don't have introspection test for Actions. Also not on the previous format. I will add them. Next thing on my list.